app/models/pusher.rb

```diff
@@ -128,6 +128,7 @@ def after_write
       Mailer.gem_pushed(user.id, @version_id, notified_user.id).deliver_later
     end
     Indexer.perform_later
+    UploadVersionsFileJob.perform_later
     ReindexRubygemJob.perform_later(rubygem:)
     GemCachePurger.call(rubygem.name)
```

Comment:
Does this need any changes to be delayed until after the background job to update the compact index has completed?

Reply:
Not right now -- when we change to serving from S3 by default, we probably want to remove the versions purge from `GemCachePurger` and only call it in `UploadVersionsFileJob`, but since we're explicitly only purging the S3 file in `UploadVersionsFileJob` (and also not serving it by default, see the VCL PR), I think we're good for now